package.json

```diff
@@ -108,6 +108,7 @@
     "core-js": "^2.4.1",
     "crypto-js": "^3.1.9-1",
     "eslint-scope": "^4.0.0",
+    "grunt": "^1.0.3",
```

Comment:
Why would you add grunt ? Prebid uses gulp

Reply:
grunt removes